doc/concept/granular-certificates/commands/withdraw.md

```diff
@@ -6,7 +6,7 @@
 > [!IMPORTANT]
 > This command has **NOT** been implemented yet.
 
-There is a number of outstading issues to discuss.
+There is a number of outstanding issues to discuss in terms of this command's functionality.
 
 - What should happen with claims that uses a certificate?
-- It the a period within it is allowed?
+- It there a period within which it is allowed?
```

Comment:
Absolutely a todo - come to think about it we should definately add a github action raising issues for TODOs in the comments @lauranolling

Reply:
Great idea @wisbech. Added a feature request for this: #68